ansible/roles/test/files/ptftests/vrf_test.py

```diff
@@ -68,13 +69,13 @@ def setUp(self):
         """
         super(FwdTest, self).setUp()
         self.test_balancing = self.test_params.get('test_balancing', False)  # default not to test balancing
-        self.fwd_info = self.test_params.get('fwd_info', None)
+        self.fib_info_files = self.test_params.get('fib_info_files', None)
```

Comment:
Why changing this attribute? I don't see any place use more than 1 file? Do you have plan to use more files?

Reply:
Vrf tests inherit **FibTest** which now requires such params as: 'router_macs', 'fib_info_files', 'ptf_test_port_map'. 
I don't plan to use more files here, but FibTest will fail if it will receive **fwd_info** parameter instead of expected **fib_info_files**.

Reply:
@yxieca can you please take a look? Thank you in advance!